relay-replays/src/snapshots/relay_replays__recording__tests__scrub_pii_key_based_edge_cases.snap

```diff
@@ -0,0 +1,45 @@
+---
+source: relay-replays/src/recording.rs
+expression: scrubbed
+---
+[
+  {
+    "data": {
+      "payload": {
+        "data": {
+          "method": "POST [ip]",
+          "request": {
+            "body": {
+              "api_key": "[Filtered]",
+              "events": [],
+              "options": {
+                "min_id_length": 1,
+              },
+            },
+            "passwd": "[Filtered]",
+            "size": 2949,
+            "whatever": "scrub this IP: [ip]",
+          },
+          "response": {
+            "body": {
+              "code": 200,
+              "events_ingested": 5,
+              "payload_size_bytes": 2949,
+              "server_upload_time": 1680009712652,
+            },
+            "size": 94,
+          },
+          "statusCode": 200,
+          "token": "[Filtered]",
```

Comment:
`token` is also in [PASSWORD_KEY_REGEX](https://github.com/getsentry/relay/blob/258783fc348c7920cd3651734688ba95d12f5968/relay-general/src/pii/regexes.rs#L275-L279).